File: library/src/main/java/com/keelim/aio/OpenItem.java
package com.keelim.aio;

public class OpenItem {
    private String sector;
    private String desc;

    public OpenItem(String sector, String desc) {
        this.sector = sector;
        this.desc = desc;
    }

    public String getSector() {
        return sector;
    }

    public void setSector(String sector) {
        this.sector = sector;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }
}

File: library/src/main/java/com/keelim/aio/OpenItem.kt
package com.keelim.aio

class OpenItem {
    constructor(sector: String?, desc: String?) {
        this.sector = sector
        this.desc = desc

    }

    constructor() {
        sector = null
        desc = null

    }

    var sector: String? = null
    var desc: String? = null

}